libs/src/oracle2/services/assertions/gql/factory.ts

```diff
@@ -1,5 +1,6 @@
 import type { ChainId } from "@shared/types";
 import { parseAssertionGraphEntity } from "@shared/utils";
+import type { Address } from "wagmi";
```

Comment:
this address type is `0x${string}`. It's what wagmi wants in most cases and its more strict which makes sense to me. I've updated other types to use it as well for compatibility.